backend-api/src/main/java/pms/infrastructure/FeatureFlagsConfig.java

```diff
@@ -0,0 +1,41 @@
+package pms.infrastructure;
+
+import com.typesafe.config.ConfigFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import pms.PetMgtSystemApplication;
+
+import java.util.HashMap;
+
+@Component
+public class FeatureFlagsConfig {
+    private static final Logger LOGGER= LoggerFactory.getLogger(PetMgtSystemApplication.class);
+
+    private HashMap<String, Boolean> featureFlags = new HashMap<>(1);
+
+    public FeatureFlagsConfig() {
+        var config = ConfigFactory.load();
+        this.featureFlags.put(FeatureFlagsConstants.ACTIVATE_CASSANDRA_RW, config.getBoolean("featureFlags."+FeatureFlagsConstants.ACTIVATE_CASSANDRA_RW));
```

Comment:
We should have constants for the complete "path" instead of doing string concatenation operations here. Also is it really necessary to copy this information from the config to the `HashMap`? We could just obtain the value from the config directly

Reply:
I thought a bit about this, I also don't like this approach. 

Regarding the constants I agree, the hashmap would be used to load this config once and set them once with the toggle name as a key.

I will think a  bit more and how to improve this.

Reply:
I removed this code since it wasn't needed.